hoomd/md/pair/pair.py

```diff
@@ -110,6 +110,13 @@ class Pair(force.Force):
 
         Type: `TypeParameter` [`tuple` [``particle_type``, ``particle_type``],
         `float`])
+
+    .. py:attribute:: mode
+
+        *mode*, *optional*: defaults to ``none``.
+        Possible values: ``none``, ``shift``, ``xplor``
+
+        Type: ``str``
```

Comment:
```suggestion
        Type: `str`
```
Here we want a reference. 

In restructured text, single backticks make references to other classes/methods/sections/etc in the docs. In this case, `str` is a reference to the python3 documentation for `str` and is the recognized way of defining the type of a string. 

In the lines previous to this your usage of double backticks is correct. Double backticks indicate the contents should be rendered in a "code" font and background and should be used for defined values, especially those readers might copy and paste into code.